src/scripts/clipperUI/components/modeButton.tsx

```diff
@@ -1,19 +1,22 @@
+import {Constants} from "../../constants";
 import {Localization} from "../../localization/localization";
-
-import {ComponentBase} from "../componentBase";
 import {ClipMode} from "../clipMode";
+import {ComponentBase} from "../componentBase";
+import {PropsForAriaGrouping} from "./ariaGroupingProps";
 
-export interface ModeButtonProps {
+export interface PropsForModeElementNoAriaGrouping {
 	imgSrc: string;
 	label: string;
 	myMode: ClipMode;
-	tabIndex: number;
 	selected?: boolean;
+	tabIndex?: number;
 	onModeSelected: (modeButton: ClipMode) => void;
 	tooltipText?: string;
 }
 
-class ModeButtonClass extends ComponentBase<{}, ModeButtonProps> {
+export interface PropsForModeButton extends PropsForModeElementNoAriaGrouping, PropsForAriaGrouping { }
```

Comment:
Rename this to ModeButtonsProps to match the naming convention we use for all props